_docs_integrate/delete-attributes.md

```diff
@@ -91,12 +93,13 @@ The same is done for all predecessors of the peer shared Attribute.
 Then, the appropriate [DeleteAttributeAcceptResponseItem]({% link _docs_integrate/data-model-overview.md %}#deleteattributeacceptresponseitem) is generated and sent back in the [Reponse]({% link _docs_integrate/data-model-overview.md %}#response) to the Sender of the Request.
 There, the `deletionInfo` of the corresponding own shared Attribute and its predecessors is set with `deletionStatus` `"ToBeDeletedByPeer"` and the `deletionDate` received in the Response.
 
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/a738fd76-2fe0-4e3e-a0fd-bec86b3b7939" id="ZGDlhHzeKlb-"></iframe></div>
```

Comment:
- The `deletionInfo` of the own shared Attribute and the predecessors is not set anymore when the Response is received, but when the Request is sent.
- It would be cool if you could add at least that the `deletionStatus` of the own shared Attribute and the corresponding predecessors is set to `"DeletionRequestSent"` after the Request was sent in the diagram. Maybe also add the flow, where the `deletionStatus` is set to `"DeletionRequestRejected"`.